docs/projects/version-control/unsupported-config-as-code-scenarios.md

```diff
@@ -0,0 +1,89 @@
+
+:::warning
+The first version of Configuration as Code only stores the deployment process in the git repo. It does not store variables or runbook processes. Once variables are added, the amount of scaffolding data will increase.  
+:::
+
+Typically, having two instances is a result of splitting an Octopus Deploy instance by environment (one instance has Dev/Test the other has Staging/Prod), by Tenant (one instance has test tenants, the other has customers), or both. Pointing multiple instances at the same will only work if all instances _are the exact same forever_.
+
+### Octopus Terraform Provider
+
+Use the [Octopus Terraform Provider](https://registry.terraform.io/providers/OctopusDeployLabs/octopusdeploy/latest/docs) to keep multiple instances in sync. Use Terraform's [variable functionality](https://www.terraform.io/language/values/variables) to manage the differences between the instances. For example, have a variable for environment scoping. One instance populates the environment list with "Test" while the other populates it with **Staging** and **Production**.
+
+:::warning
+You will still need a process to keep step templates in sync.
+:::
+
+The downside to this approach is you'll be unable to use the Octopus Deploy UI to manage your deployment processes. In addition, you'll need to convert your existing deployment process into Terraform manually. The files generated by Configuration as Code has a similar syntax as the Terraform provider, but it is not a 1:1 match.
+
+### Separate Folders for each instance
+
+Another alternative is each instance points to a unique folder in the same GitHub repo. For example, if you had a Developer instance and a Production instance.
+
+- .octopus/project-a/dev-instance
+- .octopus/project-a/production instance
+
+Use a file diff tool, Beyond Compare, Meld, WinMerge, or KDiff, to manually copy specific changes between the two directories. Any instance-specific configuration, such as environment scoping, worker pools, or feeds, would be excluded.
+
+:::warning
+You will still need a process to keep step templates in sync.
+:::
+
+The downside to this approach is it is a manual process and prone to error.  
+
+## Project Templating
+
+The configuration as code feature is not designed to allow two or more projects on the same space to point to the same folder. We've seen our users attempt to use Configuration as Code as project templating. This scenario is unsupported.
+
+While it may work for the first couple of projects, all the required scaffolding data is there; projects will have subtle differences. Those differences include:
```

Comment:
```suggestion
While it may work for the first couple of projects, all the necessary scaffolding data is there; projects will have subtle differences. Those differences include:
```